scripts/check_no_panic_in_production.py

```
@@ -0,0 +1,169 @@
+    pending_test_attr = False  # True if previous non-blank, non-attr line was a test attribute
+
+    violations: list[str] = []
+
+    for lineno, raw_line in enumerate(lines, start=1):
+        stripped_line = strip_comments_and_strings(raw_line)
+        bare = raw_line.strip()
+
+        if not bare or bare.startswith("//"):
+            continue
+
+        is_attr_line = bare.startswith("#[") or bare.startswith("#![")
```

Ruff (0.15.10)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)